src/test/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionListTest.java

```diff
@@ -79,4 +99,113 @@
         });
     }
 
+    @Test public void forceLoadRunningExecutionsAfterRestart() throws Throwable {
```

Comment:
Any associated `@Issue` at this point?

Reply:
Not yet, I will double-check for existing issues tomorrow and file something linked to JENKINS-43587.

Reply:
I filed [JENKINS-67164](https://issues.jenkins.io/browse/JENKINS-67164). I only added `@Issue("JENKINS-67164")` to the other test though, since this one just makes sure that `FlowExecutionList` resumes Pipelines after Jenkins restarts and isn't directly related to the issue.